__tests__/app.test.js

```diff
@@ -164,5 +159,74 @@ describe("post/api/articles/:articleid/comments", () => {
       });
   });
 });
+test("returns 404 error if article id not a number", () => {
+  const comment = {
+    username: "icellusedkars",
+    body: "Great article!",
+  };
+  return request(app)
+    .post("/api/articles/alex/comments")
+    .send(comment)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body).toEqual({ msg: "404, article not found" });
+    });
+});
 
+test("returns 404 error if no comment given", () => {
+  const comment = {
+    username: "icellusedkars",
+  };
+  return request(app)
+    .post("/api/articles/alex/comments")
+    .send(comment)
+    .expect(404)
+    .then(({ body }) => {
+      expect(body).toEqual({ msg: "404, article not found" });
+    });
+});
 
+describe("patch/api/articles/:article_id", () => {
+  test("updates the votes of an article by given paramater, no existing votes", () => {
+    const votes = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/articles/3")
+      .send(votes)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            article_img_url: expect.any(String),
+            votes: expect.any(Number),
```

Comment:
It's important here to state exactly how many votes we're expecting here to make sure the patch has worked as intended. I'd also say you don't need to check all of the other keys.